apps/vr-tests-react-components/src/stories/Toolbar/Toolbar.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Toolbar, ToolbarProps, ToolbarButton, ToolbarDivider, ToolbarToggleButton } from '@fluentui/react-toolbar';
+import { Menu, MenuTrigger, MenuPopover, MenuList, MenuItem } from '@fluentui/react-menu';
+import {
+  TextBold24Regular,
+  TextItalic24Regular,
+  TextUnderline24Regular,
+  AlertSnooze24Regular,
+  MoreHorizontal24Filled,
+} from '@fluentui/react-icons';
+import { ComponentMeta } from '@storybook/react';
+import { withScreenerSteps } from '../../utilities';
+
+export default {
+  title: 'Toolbar Converged',
+  Component: Toolbar,
+  decorators: [story => withScreenerSteps({ story, steps: [] })],
+} as ComponentMeta<typeof Toolbar>;
+
+export const Default = (props: Partial<ToolbarProps>) => (
+  <Toolbar {...props}>
+    <ToolbarButton aria-label="Text option - Bold" appearance="primary" icon={<TextBold24Regular />} />
+    <ToolbarButton aria-label="Text option - Italic" icon={<TextItalic24Regular />} />
+    <ToolbarButton aria-label="Text option - Underline" icon={<TextUnderline24Regular />} />
+    <ToolbarDivider />
+    <ToolbarToggleButton
+      aria-label="Snooze Alert Option"
+      name="toggle"
+      value="toggle"
+      icon={<AlertSnooze24Regular />}
+    />
+    <Menu>
```

Comment:
there's no specific or special positioning or design being used by the menu for toolbar, I would rather not have it in VR tests

Reply:
I will still add a step to open it, do you think it would be worth or should we remove it?

Reply:
I don't think it's necessary, there's nothing special about this menu vs any other menu